.github/workflows/validate-and-report.yml

```diff
@@ -171,10 +173,17 @@ jobs:
           name: build_${{ matrix.version }}
           path: build/${{ matrix.version }}
       - name: Clone asset repository
+        if: matrix.version == 'us'
         uses: actions/checkout@v4
         with:
           ref: 'gh-duplicates'
           path: 'gh-duplicates'
+      - name: Clone asset repository
+        if: matrix.version == 'pspeu'
+        uses: actions/checkout@v4
+        with:
+          ref: 'gh-duplicates-psp'
```

Comment:
What if we change this into `gh-duplicates-${{ matrix.version }}`? We could rename later on `gh-duplicates` as `gh-duplicates-us` too and merge the two `Clone asset repository` steps